src/Extensions/Annotatable.php

```diff
@@ -77,9 +82,19 @@ public function setUp()
     /**
      * @param $message
      */
-    public function displayMessage($message)
+    public function displayMessage($message, $heading = false, $end = false)
```

Comment:
PHPDoc needs updating, is also missing a type for $message

Reply:
Hah, yeah. Since I worked outside the "working directory" all the automatic PHPStorm annotations are missing :/